client/src/test/omtLinkProvider.test.ts

```diff
@@ -1,60 +1,40 @@
-import { DocumentLink, commands, workspace, Uri, Position, Range } from 'vscode';
+import { DocumentLink, commands, Position, Range } from 'vscode';
 import { getDocUri, activate } from './helper';
-import { expect } from 'chai';
+import { stub } from 'sinon';
+import { DocumentLinkRequest, LanguageClient } from 'vscode-languageclient';
+
+type Case = { i: number, line: number, start: number, end: number, target: string | undefined, should: string };
 
 describe('OMTLinkProvider', () => {
     const docUri = getDocUri('one/imports.omt');
-    const expectedCount = 6;
     let actualDocumentLinks: DocumentLink[];
+    let sendRequestStub;
 
     before(async () => {
+        sendRequestStub = stub(LanguageClient.prototype, 'sendRequest')
+            .returns(Promise.resolve([
+                new DocumentLink(toRange(1, 4, 1, 19))
+            ]));
+
         await activate(docUri);
-        actualDocumentLinks = (await commands.executeCommand(
+        const temp = (await commands.executeCommand(
```

Comment:
if you use actualDocumentationLinks = await commands.executeCommand<DocumentLink[]> you dont need the cast.

Reply:
removed in the upcoming linting commit
but good to know there is a generic version